components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java

```diff
@@ -4697,4 +4701,28 @@ public static String resolveExternalConsentPageUrl(String tenantDomain) throws I
 
         return externalConsentPageUrl;
     }
+
+    /**
+     * Check whether the application should be FAPI conformant.
+     *
+     * @param clientId       Client ID of the application.
+     * @return Whether the application should be FAPI conformant.
+     * @throws OAuthClientAuthnException
+     */
+    public static String isFapiConformantApp(String clientId) throws OAuthClientAuthnException {
```

Comment:
How about returning a boolean value? Otherwise everyplace we call this method, need to convert to a boolean or compare right?